starter-code/css/main.css

```diff
@@ -6,3 +6,148 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Helvetica, sans-serif;
+  display: flex;
+  flex-direction: column;
+}
+
+.navigation {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  font-size: 1.5em;
+  font-weight: lighter;
+  align-content: space-around;
+}
+.navitem {
+  display: flex;
+}
+.navitem div {
+  padding-left: 30px;
+}
+.logo {
+  height: 50px;
+}
+
+.headbanner {
+  background-image: url("/W1D4/lab/lab-css-spotify-clone/starter-code/images/landing.jpg");
```

Comment:
this will not show up on any other computer but yours since the path you are giving to show image includes folders that only you have on your computer. Best is to write a `./` and then click enter depending on the path you want to have. This way it will not only be visible locally but also on any other computer.